layouts/partials/javascript.html

```diff
@@ -1,6 +1,7 @@
 {{ $calendar   := .Params.calendar }}
 {{ $app        := resources.Get "js/app.js" | fingerprint }}
 <script src="https://code.jquery.com/jquery-3.4.0.min.js" integrity="sha256-BJeo0qm959uMBGb65z40ejJYGSgR7REI4+CW1fNKwOg=" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/4.2.0/anchor.min.js" integrity="sha256-m1eTvwEHwmsw4+XKF7BshClVJEPwTVycveNl0CS0Mkk=" crossorigin="anonymous"></script>
```

Comment:
More so a comment than anything else, but it'd probably be worth eventually bringing these internally. Netlify has their own CDN network and the content + js would be right alongside each other than having to goto an external source.